netbox/templates/ipam/vlantranslationrule.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load render_table from django_tables2 %}
+{% load i18n %}
+
+{% block content %}
+    <div class="row">
+        <div class="col col-md-4">
+            <div class="card">
+                <h2 class="card-header">{% trans "VLAN Translation Rule" %}</h2>
+                <table class="table table-hover attr-table">
+                    <tr>
+                        <th scope="row">{% trans "Policy" %}</th>
+                        <td>{{ object.policy|placeholder }}</td>
```

Comment:
This should link to the policy. The `placeholder` filter isn't needed here because `policy` is a required field.
```suggestion
                        <td>{{ object.policy|linkify }}</td>
```